runtime/lua/vim/fs.lua

```diff
@@ -488,6 +488,8 @@ end
 --- (default: `true`)
 --- @field expand_env? boolean
 ---
+--- @field package _fast? boolean
```

Comment:
Do you plan to remove this in 0.11? 

I wonder if vim.validate() can be made faster.

Reply:
This is fine for now but maybe we can make path_resolve_dot faster in 0.11